### PR TITLE
Update eager notebooks with buttons, licenses, and change filenames

### PR DESCRIPTION
**Pull request by lamberta**

### CODE REVIEW

**Review by MarkDaoust — DISMISSED** [review]
While we've got these open can you re-save them with "private_output" so we don't get such silly diffs, and history-bloat?

**Comment by lamberta** [comment]
Thanks. Updated, PTAL

**Review by MarkDaoust — DISMISSED** [review]
That last commit doesn't look right... why revert the names?

**Comment by lamberta** [comment]
This was my attempt to remove that file from this PR.
I'm updating that file in this branch: https://github.com/lamberta/tensorflow/blob/r1.10-tutorials/tensorflow/docs_src/tutorials/_index.yaml
Which depends on this PR.

**Comment by lamberta** [comment]
Reverted my attempt to make it easier that did not make it easier. File names should be in sync now.

File: tensorflow/contrib/eager/python/examples/notebooks/eager_intro.ipynb [hunk 1]
```diff
@@ -1,27 +1,107 @@
 {
```
Comment (MarkDaoust) [inline tensorflow/contrib/eager/python/examples/notebooks/eager_intro.ipynb]: This seems to be under the wrong name still.  eager_intro --> eager_basics right?

Comment (lamberta) [reply tensorflow/contrib/eager/python/examples/notebooks/eager_intro.ipynb]: fixed

**Review by MarkDaoust — APPROVED** [review]

**Pull request merged**